spec/jobs/deposit_collection_job_spec.rb

```diff
@@ -11,6 +11,7 @@
   before do
     allow(ToCocina::Collection::Mapper).to receive(:call).and_call_original
     allow(Sdr::Repository).to receive(:accession)
+    allow(Settings.notifications).to receive(:enabled).and_return(false)
```

Comment:
Do we usually want `Settings.notifications` to be `true` when running tests, but `false` in job specs?

Reply:
Hmmm ... since only triggers emails, probably mostly want it to be false. Maybe worth a follow-up refactor to default to false for tests and only enable for tests when necessary?

Reply:
@justinlittman Sounds good. 🙂